NaviGator/gnc/navigator_thrust_mapper/navigator_thrust_mapper/thruster_map.py

```diff
@@ -104,8 +157,8 @@ def from_urdf(cls, urdf_string, transmission_suffix='_thruster_transmission'):
                 if limit != -1 and joint.limit.effort != limit:
                     raise Exception('Thruster {} had a different limit, cannot proceed'.format(joint.name))
                 limit = joint.limit.effort
-
-        return cls(names, positions, angles, ratio, limit, joints=joints)
+        limit_tuple = (limit, -limit)
+        return cls(names, positions, angles, ratio, limit_tuple, joints=joints)
```

Comment:
did you mean:
return cls(names, positions, angles, generate_linear_force_to_command(ratio), limit_tuple, joints=joints)
?